From 397570574c63691197aff948ddffd0ec1e7f8678 Mon Sep 17 00:00:00 2001
From: Thomas HAMEL <thomas-t.hamel@edf.fr>
Date: Fri, 26 Jul 2024 14:19:22 +0200
Subject: [PATCH 3/3] do not rely on package.version major attribute

This attribute is not present in the version in EL8
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0b45d63..e7087bf 100644
--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,7 @@ def get_version():
 
 
 VERSION = get_version()
-SLURM_VERSION = f"{VERSION.major}.{VERSION.minor}"
+SLURM_VERSION = str(VERSION)[:str(VERSION).find('.',3)] #remove after second dot
 
 
 def homepage(*args):
-- 
2.30.2

